src/components/balance/BalancePlasm.vue

```diff
@@ -12,6 +12,17 @@
         <ToggleMetaMask />
       </div>
     </div>
+    <div v-if="isSS58">
+      <div class="tw-grid md:tw-auto-cols-max xl:tw-grid-cols-2 tw-gap-4">
+        <Wallet v-model:isOpen="modalAccount" :wallet-name="currentAccountName" />
```

Comment:
![Screen Shot 2021-11-25 at 6 03 20 PM](https://user-images.githubusercontent.com/40356749/143480479-de807542-cead-4192-baa1-8d30cbd093bd.png)

I see two account selection dropdowns, one on top and one below the Ethereum toggle button (under `Ethereum Extension`).

Plus, the top and bottom margin does not seem to look right

Reply:
Modified. Kindly take a look again.
https://gyazo.com/2526f6058d42c8745f8d8e92935095dd